contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -124,16 +125,15 @@ contract CompatibilityFallbackHandler is TokenCallbackHandler, ISignatureValidat
                     // address() has been changed to caller() to use the implementation of the Safe
                     caller(),
                     0,
-                    internalCalldata,
+                    ptr,
                     calldatasize(),
                     /**
                      * The `simulateAndRevert` call always reverts, and
                      * instead encodes whether or not it was successful in the return
                      * data. The first 32-byte word of the return data contains the
-                     * `success` value, so write it to memory address 0x00 (which is
-                     * reserved Solidity scratch space and OK to use).
+                     * `success` value, so write it to `returnPtr`.
```

Comment:
I need help to follow this. There's no `returnPtr` pointer in the file. Doesn't it just write the return data right after the calldata copied earlier?

Reply:
That's a typo based on previous edits, will change in the next commit 👍🏾